relay-server/src/services/processor.rs

```diff
@@ -335,6 +330,16 @@ impl ProcessingGroup {
             ))
         }
 
+        // NEL items are converted to logs, but need their own processing before being merged with
+        // logs.
```

Comment:
Is there actually any processing necessary for NEL items other than converting them to logs?

Reply:
No, that *is* the processing (currently).

Reply:
I am not sure, is this confusing to you, if yes what is the confusing part?

NEL items do have their own processing step (called `process_nel`) before they are merged with logs (`process_logs`).